modules/scheduler-impl/src/main/java/org/opencastproject/scheduler/endpoint/SchedulerRestService.java

```diff
@@ -578,10 +578,13 @@ public Response getCalendarJson(
         }
       }
 
-      var result = new ArrayList<TechnicalMetadata>();
+      var result = new ArrayList<Map<String, Object>>();
       for (var event: service.search(agent, Opt.none(), Opt.none(), Opt.some(new Date()), endDate)) {
         var id = event.getIdentifier().toString();
-        result.add(service.getTechnicalMetadata(id));
+        result.add(Map.of(
+                "agent", service.getTechnicalMetadata(id),
```

Comment:
Not sure about the `agent` naming. What do you think about `meta`?

Reply:
I don't really care that much, though I don't quite understand what `meta` would mean in this context, since you then would have meta(data?) and other metadata?

Reply:
mmh ok fair enough. Still not convinced about `agent` as this does not describe the agent 😄. Maybe `episode`, `event`, `data`?

Reply:
Personally, I would prefer the first one. So just a rename.